atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/TargetedSweepEteTest.java

```diff
@@ -40,6 +44,9 @@
     private static final TableReference VALUES_TABLE =
             FACTORY.getSnapshotsStreamValueTable(null).getTableRef();
 
+    @Rule
+    public final TestRule flakeRetryingRule = new FlakeRetryingRule();
+
```

Comment:
What is this for?

Reply:
Allows me to use the `@ShouldRetry` annotation below, which retries the test 5 times and retries if it throws an `Exception` or `AssertionError` IIRC.